### PR TITLE
fix(website): Fix contrast of code snippets in light mode

### LINKED ISSUE #8227: bug(website): Light mode code snippets are hard to read
Example:
image

In https://www.cloudquery.io/docs/plugins/sources/aws/overview

### PR DESCRIPTION
**Pull request by nagaajay4**
Closes #8227

### CODE REVIEW

**Review by erezrokah — CHANGES_REQUESTED** [review]
Hi @nagaajay4 and thank you for the PR.
Should this update the styling on light mode code snippets?
I'm seeing this on the [deploy preview](https://cloudquery-kcj20fh6g-cloudquery.vercel.app/docs/plugins/sources/aws/overview):
image

**Comment by nagaajay4** [comment]
Hi @erezrokah  and thanks for pointing it out. It is working fine with new changes
ScreenShot_27-02-2023_12_31_50

**Comment by hermanschaaf** [comment]
Hi @nagaajay4, thanks for this change, the darker background color for code snippets makes it much easier to read!

I think the style change had some unintended side-effects, though. For example, on the `/integrations/aws/bigquery` page, currently the breadcrumbs look like this:

Screenshot 2023-02-27 at 08 42 58

On the preview branch they look like this:

Screenshot 2023-02-27 at 08 43 18

(The final link is a bit too light)

I also noticed that the "copy" links on the top right-hand side of code snippets are hard to see against the new background:

Screenshot 2023-02-27 at 08 45 39

Would you mind taking a look at these and seeing if there is a good solution for them?

**Comment by nagaajay4** [comment]
Hi @hermanschaaf, good observation. Thanks for letting me know. I will soon update the pr with a new solution without unintended side effects.

**Comment by nagaajay4** [comment]
Hey, @erezrokah and @hermanschaaf tried a better solution and unintended side effects are removed,(copy and toggle option links on the top right-hand side of code snippets are made clearly visible). Please let me know further suggestions or corrections.

ScreenShot_27-02-2023_20_20_31
ScreenShot_27-02-2023_20_19_12

**Review by hermanschaaf — APPROVED** [review]
Looks good to me, thanks for the fix @nagaajay4 !

File: website/styles.css [hunk 1]
```diff
@@ -10,12 +10,30 @@ body {
...
+/* Overriding the default colour schemas for light mode*/
+.light [data-language], .light .nextra-code-block>pre {
+  background-color: #0a4b6c  !important;
+}
+.light .nextra-code-block>div {
+  background-color: #0a4b6c   !important;
+  color: aliceblue !important;
+}
+.light button[title="Copy code"], button[title="Toggle word wrap"] {
+    color: aliceblue !important;
```
Comment (hermanschaaf) [inline website/styles.css:22]:
```suggestion
  color: aliceblue !important;
```

Some extra spaces here

File: website/styles.css [hunk 2]
```diff
@@ -10,12 +10,30 @@ body {
...
 }
 
+/* Overriding the default colour schemas for light mode*/
+.light [data-language], .light .nextra-code-block>pre {
+  background-color: #0a4b6c  !important;
+}
+.light .nextra-code-block>div {
+  background-color: #0a4b6c   !important;
+  color: aliceblue !important;
+}
```
Comment (hermanschaaf) [inline website/styles.css:20]:
```suggestion
.light [data-language], .light .nextra-code-block>pre {
  background-color: #0a4b6c !important;
}
.light .nextra-code-block>div {
  background-color: #0a4b6c !important;
  color: aliceblue !important;
}
```

**Review by erezrokah — APPROVED** [review]
Thanks for all the follow up @nagaajay4 🚀

**Pull request merged**